terraform/paas/resources.tf

```diff
@@ -0,0 +1,69 @@
+resource cloudfoundry_app teacher-training-api {
+  name         = var.app.name
+  space        = data.cloudfoundry_space.space.id
+  docker_image = var.app.docker_image
+  strategy     = "blue-green-v2"
+
+  service_binding {
+    service_instance = cloudfoundry_service_instance.postgres.id
+  }
+
+  service_binding {
+    service_instance = cloudfoundry_service_instance.redis.id
+  }
+
+  routes {
+    route = cloudfoundry_route.teacher-training-api-route.id
+  }
+
+  environment = {
+#     APPINSIGHTS_INSTRUMENTATIONKEY                   = var.APPINSIGHTS_INSTRUMENTATIONKEY
+    RAILS_ENV                                        = var.app_env.RAILS_ENV
+    WEBSITE_SLOT_POLL_WORKER_FOR_CHANGE_NOTIFICATION = "0"
+    SETTINGS__LOGSTASH__HOST                         = var.SETTINGS__LOGSTASH__HOST
+    SETTINGS__LOGSTASH__PORT                         = var.app_env.SETTINGS__LOGSTASH__PORT
+    RAILS_SERVE_STATIC_FILES                         = var.app_env.RAILS_SERVE_STATIC_FILES
+  }
+}
+
+resource cloudfoundry_route teacher-training-api-route {
+  domain   = data.cloudfoundry_domain.local.id
+  space    = data.cloudfoundry_space.space.id
+  hostname = var.app.hostname
+}
+
+resource "cloudfoundry_service_instance" "postgres" {
+  name         = var.paas_postgres_name
+  space        = data.cloudfoundry_space.space.id
+  service_plan = data.cloudfoundry_service.postgres.service_plans["small-11"]
+  json_params  = "{\"enable_extensions\": [\"postgis\"] }"
```

Comment:
Do we need postgis? I don't think we use it. According to the schema https://github.com/DFE-Digital/teacher-training-api/blob/master/db/schema.rb#L16 we need
```
 16   enable_extension "pg_buffercache"
 17   enable_extension "pg_stat_statements"
 18   enable_extension "plpgsql"
```
although I'd have to check if they are addons or part of the postgres packages

Reply:
Removed, thanks